setup.py

```diff
@@ -10,7 +10,7 @@
 from setuptools import find_packages, setup
 
 requirements = [
-    'compas>=1.3,<2.0',
+    'compas>=1.6,<2.0',
```

Comment:
so i guess this should be
```suggestion
    'compas>=1.7,<2.0',
```

Reply:
indeed